src/Workspaces/Core/Portable/Options/OptionSet+AnalyzerConfigOptionsImpl.cs

```diff
@@ -43,7 +43,9 @@ public override bool TryGetValue(string key, [NotNullWhen(true)] out string? val
                     return false;
                 }
 
-                value = storageLocation.GetEditorConfigStringValue((OptionKey)optionKey, _optionSet);
+                var storageLocation = (IEditorConfigStorageLocation)option.StorageLocations.Single();
```

Comment:
should option2 have an editorconfig storage location as a field?  Maybe this changes in a followup commit

Reply:
Done in subsequent commit.